metta/metta_io.py

```diff
@@ -21,7 +21,8 @@ def archive_train_test(train_config,
                        df_test,
                        directory='.',
                        format='hd5',
-                       overwrite=False):
+                       overwrite=False,
+                       write_index=True):
```

Comment:
The default behavior of pandas is the write the index to a csv file. Are there cases when we would not want that? I am thinking about when pandas adds an arbitrary numerical index.